dataprofiler/tests/profilers/test_numeric_stats_mixin_profile.py

```diff
@@ -288,7 +286,7 @@ def test_histogram_bin_error(self):
 
         # Sum of errors should be difference of each input value to midpoint of bin squared
         # bin_midpoints = [2, 6, 10, 14]   ids = [1, 1, 2, 3, 3, 4]
-        assert sum_error == (2-0)**2 + (2-3)**2 + (6-5)**2 + (10-9)**2 + (10-11)**2 + (17-14)**2
+        assert sum_error == (2 - 0) ** 2 + (2 - 3) ** 2 + (6 - 5) ** 2 + (10 - 9) ** 2 + (10 - 11) ** 2 + (17 - 14) ** 2
```

Comment:
first, let's break this into multi line to be < 80 chars. Additionally, if we are changing this, maybe we should do a `self.assertEqual`